docs/do-not-merge.md

```diff
@@ -30,29 +32,37 @@ As part of this feature, there are new API endpoints available to interact with
 
 - Get query results with `GET /sql/statements/:queryID`
 
-[14416](https://github.com/apache/druid/pull/14416)
-[14512](https://github.com/apache/druid/pull/14512)
-[14527](https://github.com/apache/druid/pull/14527)
+[#14416](https://github.com/apache/druid/pull/14416)
 
-## Java 17 support
+[#14512](https://github.com/apache/druid/pull/14512)
 
-Druid now fully supports Java 17.
-[14384](https://github.com/apache/druid/pull/14384)
+[#14527](https://github.com/apache/druid/pull/14527)
 
-## Array column types
+ADD LINK TO DOCS
 
-Array column types are now generally available.
-
-For more information about this feature, see the [26.0.0 release notes for array column types](https://github.com/apache/druid/releases#26.0.0-highlights-auto-type-column-schema-%28experimental%29).
+WILL TO JAZZ IT UP 
 
 ## Schema auto-discovery
 
 Schema auto-discovery is now generally available.
 
 For more information about this feature, see the [26.0.0 release notes for Schema auto-discovery](https://github.com/apache/druid/releases#26.0.0-highlights-auto-type-column-schema-%28experimental%29-schema-auto-discovery-%28experimental%29).
 
+## Array column types - COMBINE WITH SCHEMA AUTO-DISCOVERY
+
+As part of Schema auto-discovery, array column types are now generally available. 
+
+- THERE ARE CAVEATS MUST USE AUTO TYPE COLUMN INDEXER FROM SCHEMA AUTO-DISCOVERY
+- GET CAVEATS FROM CLINT
+
+
+For more information about this feature, see the [26.0.0 release notes for array column types](https://github.com/apache/druid/releases#26.0.0-highlights-auto-type-column-schema-%28experimental%29).
+
+
 ## Smart segment loading
 
+INSTEAD OF PRIORITIZING THINGS EQUALLY; WE PRIORITIZE NEW SEGMENTS THAT ARE UNDERREPLICATED
```

Comment:
**Suggested content for this section:**

The Coordinator is now much more stable and user-friendly. In the new `smartSegmentLoading` mode, it dynamically computes values for several configs which maximize performance.

The Coordinator can now prioritize load of more recent segments and segments that are completely unavailable over load of segments that already have some replicas loaded in the cluster. It can also re-evaluate decisions taken in previous runs and cancel operations that are not needed anymore. Moreoever, move operations started by segment balancing do not compete with the load of unavailable segments thus reducing the reaction time for changes in the cluster and speeding up segment assignment decisions.